src/qt/rpcconsole.cpp

```diff
@@ -658,6 +659,7 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
         ui->blocksDir->setText(model->blocksDir());
         ui->startupTime->setText(model->formatClientStartupTime());
         ui->networkName->setText(QString::fromStdString(Params().NetworkIDString()));
+        ui->scrollArea->horizontalScrollBar()->setStyleSheet("QScrollBar {height:0px;}");
```

Comment:
This should be ok to do since #293 was merged. Instead of changing the stylesheet, you should implement this suggestion: https://github.com/bitcoin-core/gui/pull/324/files#r630627901 

You should change this line as so:

```diff
-   ui->scrollArea->horizontalScrollBar()->setStyleSheet("QScrollBar {height:0px;}");
+   ui->scrollArea->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
```